api/app/controllers/concerns/spree/api/v2/storefront/order_concern.rb

```
@@ -39,10 +39,7 @@ def serialized_current_order
           end
 
           def serialize_order(order)
-            Spree::Deprecation.warn(<<-DEPRECATION, caller)
-              `OrderConcern#serialize_order` is deprecated and will be removed in Spree 5.0.
-              Please use `serializer_resource` method
-            DEPRECATION
+            Spree::Deprecation.warn('OrderConcern#serialize_order is deprecated and will be removed in Spree 6.0. Please use `serializer_resource` method')
```

RuboCop (1.75.5)

[convention] Line is too long. [155/150]

(Layout/LineLength)